core/domain/user_services_test.py

```diff
@@ -1318,6 +1603,242 @@ def test_record_user_started_state_translation_tutorial(self):
         self.assertTrue(
             user_settings.last_started_state_translation_tutorial is not None)
 
+    def test_get_human_readable_user_ids(self):
+        auth_ids = ['user1', 'user2', 'user3']
+        user_emails = [
+            '1user@example.com',
+            '2user@example.com',
+            '3user@example.com']
+        user_ids = []
+
+        for i, auth_id in enumerate(auth_ids):
+            user_ids.append(user_services.create_new_user(
+                auth_id, user_emails[i]).user_id)
+
+        user_services.set_username(user_ids[0], 'username1')
+        user_services.mark_user_for_deletion(user_ids[1])
+
+        user_settings3 = user_services.get_user_settings(user_ids[2])
+
+        usernames = [
+            'username1',
+            user_services.LABEL_FOR_USER_BEING_DELETED,
+            (
+                '[Awaiting user registration: %s]' %
+                user_settings3.truncated_email
+            )
+        ]
+
+        self.assertEqual(
+            usernames,
+            user_services.get_human_readable_user_ids(user_ids)
+        )
+
+    def test_get_human_readable_user_ids_no_user_raises_error(self):
```

Comment:
Perhaps you can add a word here to make it scan.

Reply:
Done